code/api/utility.py

```diff
@@ -1,10 +1,63 @@
 from django.http.response import HttpResponseBadRequest
 from django.core.paginator import Paginator
+from django.shortcuts import get_object_or_404
+
+from socialDistribution.models import *
 
 # https://docs.djangoproject.com/en/3.2/topics/pagination/ - Pagination
 def getPaginated(data, page, size):
     p = Paginator(data, size)
     try:
         return p.page(page)
     except:
-        return []
\ No newline at end of file
+        return []
+    
+def makePost(author_id, data):
```

Comment:
Comment for this function would be nice, especially since this function does a lot (gets an author, finds an inbox post, adds the post to inbox)